src/main/java/com/revature/repos/UserRepo.java

```diff
@@ -40,10 +40,10 @@
 	public List<User> findByStatus();
 	
 	//Query to get all dropped associates in the last week
-	@Query(value="select * from user_service.sms_users left join user_service.status_history on users_id = sms_user_id where status_start between now() - INTERVAL '168 HOURS' and now()", nativeQuery=true)
```

Comment:
We should leave the schema name alone. This could break the deployed environment.

Reply:
The same as above